### PR TITLE
Updated_The_Authentication_Attacks_Workbooks

### PR DESCRIPTION
**Pull request by hemantvish97**
Updated the Authentication Attacks Workbooks(15).

### CODE REVIEW

**Review by aman1808 — CHANGES_REQUESTED** [review]

File: workbooks/Basic Security Monitoring/Authentication Attacks/abnormal_ssh_login_attempts_for_a_user_e192a064480b46f78350cc3fd080f910.yml [hunk 1]
```diff
@@ -43,47 +39,51 @@ blocks:
   DetectionName: Abnormal SSH Login Attempts for a User
   DetectionTactic: Initial Access
   DetectionTechnique: External Remote Services
-  DetectionScore: '7'
-  DetectionSeverity: High
+  DetectionScore: '5'
+  DetectionSeverity: Medium
   DetectionConfidence: High
   SuspectHost: $SrcIP
```
[the remark this answers is not in the document]
Comment (hemantvish97) [reply workbooks/Basic Security Monitoring/Authentication Attacks/abnormal_ssh_login_attempts_for_a_user_e192a064480b46f78350cc3fd080f910.yml:45]: We have tested and found the output is "SrcIP" So, we have changed the value in the workbook as per the output.

File: workbooks/Basic Security Monitoring/Authentication Attacks/abnormal_vpn_login_attempts_for_a_user_5e8246d661064bdb9e66e171773999db.yml [hunk 2]
```diff
@@ -43,53 +39,57 @@ blocks:
   DetectionName: Abnormal VPN Login Attempts for a User
   DetectionTactic: Initial Access
   DetectionTechnique: External Remote Services
-  DetectionScore: '7'
-  DetectionSeverity: High
+  DetectionScore: '5'
+  DetectionSeverity: Medium
   DetectionConfidence: High
   SuspectHost: $SrcIP
```
[the remark this answers is not in the document]
Comment (hemantvish97) [reply workbooks/Basic Security Monitoring/Authentication Attacks/abnormal_vpn_login_attempts_for_a_user_5e8246d661064bdb9e66e171773999db.yml:45]: We have tested and found the output is "SrcIP" So, we have changed the value in the workbook as per the output.

File: workbooks/Basic Security Monitoring/Authentication Attacks/login_failure_from_expired_account_80b3d7f21d66406dad433180349a3bca.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+path: /Basic Security Monitoring/Authentication Attacks
+description: This detection rule, unauthorized login failures are identified from user accounts that have expired, bolstering system security by preventing access from outdated credentials.
+type: standard
+author: DNIF
+stage: ''
+profile: ''
+entity_type: workbooks
+blocks:
+- type: dql
+  query: 'stream=AUTHENTICATION where action=''LOGIN'' and status=''FAILED'' and reason=''ACCOUNT_EXPIRED''
```
[the remark this answers is not in the document]
Comment (hemantvish97) [reply workbooks/Basic Security Monitoring/Authentication Attacks/login_failure_from_expired_account_80b3d7f21d66406dad433180349a3bca.yml:12]: Done with the changes

File: workbooks/Basic Security Monitoring/Authentication Attacks/login_failure_from_expired_account_80b3d7f21d66406dad433180349a3bca.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- type: signal
+  DetectionName: Login Failure From Expired Account
+  DetectionTactic: Initial Access
+  DetectionTechnique: Valid Accounts
+  DetectionScore: '4'
+  DetectionSeverity: Medium
+  DetectionConfidence: Medium
+  SuspectUser: user
+  SuspectHost: srcip
+  raw_event_query: 'stream=AUTHENTICATION where action=''LOGIN'' and status=''FAILED''
```
[the remark this answers is not in the document]
Comment (hemantvish97) [reply workbooks/Basic Security Monitoring/Authentication Attacks/login_failure_from_expired_account_80b3d7f21d66406dad433180349a3bca.yml:23]: Done with the changes

File: workbooks/Basic Security Monitoring/Authentication Attacks/multiple_successful_login_from_different_country_by_single_user_6d800284e91a479f92f4cb1e187294d6.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+type: standard
+author: DNIF
+stage: null
+profile: null
+entity_type: workbooks
+blocks:
+- type: dql
+  query: stream=AUTHENTICATION where action='LOGIN' and status='PASSED' | duration
+    30m | select user, distinct_count(srccn) as CountSrcCN | groupby user | having
+    CountSrcCN > 3
```
[the remark this answers is not in the document]
Comment (hemantvish97) [reply workbooks/Basic Security Monitoring/Authentication Attacks/multiple_successful_login_from_different_country_by_single_user_6d800284e91a479f92f4cb1e187294d6.yml:14]: Done with the changes

**Review by aman1808 — APPROVED** [review]

**Pull request merged**